spec/features/community_spec.rb

```diff
@@ -50,25 +50,20 @@
     let!(:collection3) do
       Collection.create( title: 'Test Collection 3') do |c|
         c.apply_depositor_metadata(jill.user_key)
-        c.is_community = false 
+        c.is_community = false
         c.is_official = true
         c.is_admin_set = false
         c.belongsToCommunity = [community2.id]
       end
     end
 
-    it 'should render quickly' do
-      expect( Benchmark.realtime { visit "/communities" }).to be < 5 
-    end
-
-
     it 'should have all communities and collections' do
-      visit "/communities" 
+      expect( Benchmark.realtime { visit "/communities" }).to be < 5 # should render quickly
```

Comment:
We should condense our tests more often! Don't need two examples for this when we are just testing the same thing pretty much (especially with lots of setup code! Database IO is slow with fedora!). This tiny change halfs this test speed execution.